local/src/client/verifier-client.cc

```diff
@@ -288,11 +288,32 @@ ClientReplayFileHandler::ssn_open(YAML::Node const &node)
           (pp_version_node.Scalar() == "1" || pp_version_node.Scalar() == "2"))
       {
         errata.note(S_DIAG, "Enabling PROXY protocol for this session.");
-        // set proxy protocol header version as specified
-        _ssn->pp_version =
-            (pp_version_node.Scalar() == "1") ? ProxyProtocolVersion::V1 : ProxyProtocolVersion::V2;
+        // set the PROXY protocol version as specified
```

Comment:
Let's favor complete sentence comments (start with capital letter, end with punctuation).

Reply:
Reformatted the comments